### ISSUE TITLE
Unity Export - IE11 no longer works

### ISSUE THREAD

**Issue by mreimer01** [BabylonJS/Exporters#230]
-Unity version 2018.1.5f1
-Babylon export version 3.3.0 - alpha 3.1

IE 11 does not work with the latest exporter. Exporting the project files and viewing using localhost only displays the circular spinner loader. I have the same result when hosting the files on a web server. The index.html file loads and the circular spinner is visible but the babylon loader never displays. 

Google Chrome, Edge and Firefox do not have this issue. 

![ie11](https://user-images.githubusercontent.com/23662406/41568627-ac275a86-7334-11e8-9f34-97a3ae85d3bb.jpg)

**Comment by MackeyK24** [comment]
Lets see the console output

**Comment by mreimer01** [comment]
HTML1300: Navigation occurred.
index.html
BJS - [21:59:38]: Babylon.js toolkit plugins registered
SCRIPT1002: Syntax error
index.html (104,38)
SCRIPT1002: Syntax error
index.html (181,56)

**Comment by MackeyK24** [comment]
Dunno, cant tell from that.

But if you package up the scene and send it to me i will try take a look at
it for you

**Comment by mreimer01** [comment]
Thanks, for looking. Its a basic test scene with light, camera and cylinder primitive. Let me know if something is found.
[Babylon Test Scene.zip](https://github.com/BabylonJS/Exporters/files/2114914/Babylon.Test.Scene.zip)

**Comment by MackeyK24** [comment]
I found the the problem(s).

First... IE11 does not support => functions. I had to change game.html to use function() instead of ()=>{}

Second... For some reason IE11 does NOT like the stats.js... Get a Script Error on the Canvas Context DrawImage. I will see if there is anything to be done about that one. If not, Im afraid we will have to PULL ie11 as a supported browser (until we get can stats.js running, seeing how i have a dependency on that lib for the build-in debug info of the toolkit)
